### PR TITLE
Add a new apparmor helper

### PR DESCRIPTION
**Pull request by dosaboy**
Resolves: #750

### CODE REVIEW

File: hotsos/core/host_helpers/apparmor.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            if name in profiles['profiles']:
+                return mode
+
+    @property
+    def profiles_enforce(self):
+        return self.profiles.get('enforce', {}).get('profiles', [])
+
+    @property
+    def profiles_complain(self):
+        return self.profiles.get('complain', {}).get('profiles', [])
```
Comment (brianphaley) [inline hotsos/core/host_helpers/apparmor.py:69]: I also see 'kill' and 'unconfined' profiles should we worry about those?

File: hotsos/core/host_helpers/apparmor.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+)
+from hotsos.core.factory import FactoryBase
+from hotsos.core.host_helpers.cli import CLIHelperFile
+
+
+class AAProfile(object):
+
+    def __init__(self, name):
+        self.name = name
+        self.mode = ApparmorHelper().get_profile_mode(name)
```
Comment (brianphaley) [inline hotsos/core/host_helpers/apparmor.py:18]: There is a 'processes' section in apparmor_status should that be added?

Comment (dosaboy) [reply hotsos/core/host_helpers/apparmor.py:18]: i dont have a current need for it (was adding to help with https://github.com/canonical/hotsos/pull/724) but we can always extend later if needs be.

File: hotsos/core/host_helpers/apparmor.py (hunk 1 above)
Comment (dosaboy) [reply hotsos/core/host_helpers/apparmor.py:69]: sure why not better to have them than not

Comment (dosaboy) [reply hotsos/core/host_helpers/apparmor.py:69]: actually i think unconfined is for processes not profiles so i think the only two statuses you can have are "complain" and "enforce"

**Pull request merged**